### DISCUSSION TITLE
How to receive passing value in the create pages?

### DISCUSSION THREAD

**Discussion by jigujigu** [filamentphp/filament#6314] [Help] [answered]
In my PickupRequestResource on columns table section.
I have a link for goto shipments.create resources.

```
Tables\Columns\TextColumn::make('title')
	->url(function (PickupRequest $record): string {
		 return route('filament.resources.pilgrims.create', ['pickupRequest' => $record->pickup_address]);
	})
```
					
How I can get the pickup_address as default value for some field in my shipments.create or in my shipment ressource ???

Thank you.

**Comment by jigujigu** [comment]
anyone can help me ?
Or any Ide to better way ?
thanks

**Reply by danharrin** [reply, marked as answer]
Using `request()->query('pickupRequest')`?
